recipe/meta.yaml

```diff
@@ -7,11 +7,15 @@ package:
 source:
   fn: rios-{{ version }}.tar.gz
   url: https://bitbucket.org/chchrsc/rios/downloads/rios-{{ version }}.tar.gz
-  md5: c6d370262a2630ef81deb53b88e5d184
+  sha256: 59fc71a3cfdc2c6de47fd38bc999bdb6de48ce882c5a680dac97118bf86b486f
 
 build:
-  number: 4
+  number: 0
```

Comment:
Do you want to try `noarch: python` here? Seems like `rios` is a good candidate for it.

Reply:
Indeed! Good thought